src/Redis/Initializer.php

```diff
@@ -128,7 +128,14 @@ public static function registerCommandsRediska(): void
 
     public static function getRediSearchVersion(Client $client): ?string
     {
-        $modules = $client->executeRaw('module', 'list') ?? [];
+        try {
+            $modules = $client->executeRaw('module', 'list') ?? [];
+        } catch (\Throwable $exception) {
+            if (strpos($exception->getMessage(), 'unknown command') === false) {
+                throw $exception;
+            }
+            $modules = [];
```

Comment:
It can return `null` directly, because the `foreach` will have nothing to run
```suggestion
            return null;
```